include/tgfx/gpu/RuntimeEffect.h

```diff
@@ -28,6 +28,20 @@ namespace tgfx {
  */
 class RuntimeEffect {
  public:
+  /**
+   * Generates a unique programID for the effect.
+   * The programID MUST be generated using NextProgramID to avoid conflicts with other effects.
```

Comment:
Generates a globally unique program ID for the custom runtime effect. This ID is used by the RuntimeEffect::programID() method.